index.js

```diff
@@ -4,7 +4,7 @@ const { format } = require('util')
 
 function build () {
   const codes = {}
-  const emitted = new Map()
+  const emitted = {}
```

Comment:
This is going backward. We should use a `Map`.

Reply:
Please see my next comment.

Reply:
You seem to be optimizing for nothing. Please show where this module is sitting in a hot path where such an infinitesimal difference matters.